### PR TITLE
added page and logo for warmup4IE thermostat

### PR DESCRIPTION
**Pull request by alex-0103**
**Description:**
new documentation page for warmup4IE thermostat, see PR link below.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21144

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/climate.warmup4ie.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+The `warmup4ie` climate platform let you control Warmup 4IE thermostats from Home Assistant.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: warmup4ie
+    name: YOUR_DESCRIPTION
```
Comment (klaasnicolaas) [inline source/_components/climate.warmup4ie.markdown]: You can remove this line

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/climate.warmup4ie.markdown (hunk 1 above)
Comment (alex-0103) [reply source/_components/climate.warmup4ie.markdown]: Okay, Done.

File: source/_components/climate.warmup4ie.markdown [hunk 2]
```diff
@@ -33,11 +32,6 @@ To use this component, you have to setup an account at the warmup web site first
 </p>
 
 {% configuration %}
-name:
```
Comment (klaasnicolaas) [inline source/_components/climate.warmup4ie.markdown]: restore this, only deleting from the example was the intention.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/climate.warmup4ie.markdown (hunk 2 above)
Comment (alex-0103) [reply source/_components/climate.warmup4ie.markdown]: Okay, Done. Hopefully correct now...

Comment (klaasnicolaas) [reply source/_components/climate.warmup4ie.markdown]: This is good 😄

**Review by klaasnicolaas — APPROVED** [review]
This looks good for me 😉

**Comment by klaasnicolaas** [comment]
Parent PR is still open

**Comment by alex-0103** [comment]
Yes, i have reopened a pull request for the documentation page. See #9396

**Pull request closed without merging**